docker-compose.yml

```
@@ -1,11 +1,102 @@
+      - HASURA_GRAPHQL_ADMIN_SECRET=${HASURA_GRAPHQL_ADMIN_SECRET}
+    volumes:
+      - ./metadata:/hasura-metadata
+      - ./migrations:/hasura-migrations
+    depends_on:
+      - postgres
+    networks:
+      - ubi_nw
+
+  # Keycloak Service
+  keycloak:
+    image: quay.io/keycloak/keycloak:24.0.1
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 5 but found 4

(indentation)